app/services/mailchimp_importer.rb

```
@@ -0,0 +1,73 @@
+# imports our mailchimp list to our database in an idempotent process
+
+class MailchimpImporter
+  attr_reader :mailchimp, :logger
+
+  def initialize(debug: false, logger: Rails.logger)
+    if ENV['MAILCHIMP_API_KEY']
```

should we raise an error if this envvar doesn't exist? As far as I can tell from looking at this, it seems like it would raise a less clear error somewhere down the line if this conditional evaluated to false

---

good point. The env var is defined as `required` in config/initializers/required_env_vars.rb so the app won't even boot unless that env var is set. I'll take the `if` statement out entirely.